ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.5.0f3
+m_EditorVersion: 5.5.1f1
```

Comment:
Ok, the system should now throw an assert in most cases.  The one thing it will not detect is if an anchor is created or added at runtime, since it would require extra processing overhead to detect such a case.